src/algebra/category/Module/change_of_rings.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2022 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+import algebra.category.Module.basic
+
+/-!
+# Change Of Rings
+
+## Main definitions
+
+* `category_theory.Module.restrict_scalars.functor`: given rings `R, S` and a ring homomorphism
+  `R ⟶ S`, then `restrict_scalars.functor : Module S ⥤ Module R` is defined by `M ↦ M` where
+  `M : S-module` is seen as `R-module` by `r • m := f r • m` and `S`-linear map `l : M ⟶ M'` is
+  `R`-linear as well.
+-/
+
+
+namespace category.Module
+
+namespace restrict_scalars
+
+universes u₁ u₂
+
+variables {R : Type u₁} {S : Type u₂} [ring R] [ring S] (f : R →+* S)
+variable (M : Module S)
```

Comment:
I think it would be good to be explicit about universes here. Specifically, introduce a third universe variable`v` and then write `Module.{v} R` everywhere.

Reply:
done